### PR TITLE
Bump Firesim + Use ResetPulseBridge 

### PR DESCRIPTION
**Pull request by davidbiancolin**
Lots of new FireSim features have been added over the summer. This exposes them to CY dev.

I had to fix a few things in FireSim to make this work. 

See:
firesim/aws-fpga-firesim#45
firesim/firesim#827

**Type of change**: other enhancement

**Impact**: other

**Release Notes**

### CODE REVIEW

File: .circleci/run-firesim-scala-tests.sh [hunk 1]
```diff
@@ -13,13 +13,20 @@ source $SCRIPT_DIR/defaults.sh
 # call clean on exit
 trap clean EXIT
 
+# Directory locations for handling firesim-local installations of libelf/libdwarf
+# This would generally be handled by build-setup.sh/firesim-setup.sh
+firesim_sysroot=lib-install
```
Comment (jerryz123) [inline .circleci/run-firesim-scala-tests.sh:18]: Might be safer to make this a absolute path? `$LOCAL_CHIPYARD_DIR/sims/firesim/lib-install`

**Review by jerryz123 — APPROVED** [review]

File: .circleci/run-firesim-scala-tests.sh (hunk 1 above)
Comment (abejgonzalez) [reply .circleci/run-firesim-scala-tests.sh:18]: This is done in the next two lines? What's your concern?

**Review by abejgonzalez — APPROVED** [review]
LGTM

File: .circleci/run-firesim-scala-tests.sh (hunk 1 above)
Comment (jerryz123) [reply .circleci/run-firesim-scala-tests.sh:18]: oh, misread this.

**Pull request merged**